packages-exp/analytics-exp/src/public-types.ts

```diff
@@ -67,25 +67,26 @@ export interface GtagConfigParams {
    */
   'cookie_flags'?: string;
   /**
-   * If set to false, disables all advertising features with gtag.js.
+   * If set to false, disables all advertising features with `gtag.js`.
    * See {@link https://developers.google.com/analytics/devguides/collection/ga4/display-features | Disable advertising features }
    */
   'allow_google_signals?': boolean;
   /**
-   * If set to false, disables all advertising personalization with gtag.js.
+   * If set to false, disables all advertising personalization with `gtag.js`.
    * See {@link https://developers.google.com/analytics/devguides/collection/ga4/display-features | Disable advertising features }
    */
   'allow_ad_personalization_signals'?: boolean;
   [key: string]: unknown;
 }
 
 /**
- * Analytics initialization options.
+ * Firebase Analytics instance initialization options.
```

Comment:
Product-wise, we don't say "Firebase Analytics" any more -- just "Google Analytics."

https://firebase.google.com/docs/analytics

Reply:
It's referring to the service, not the product, so I changed it to `{@link Analytics}`.